giving-example/src/main/resources/templates/layout.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html
+  lang="en"
+  xmlns:th="http://www.thymeleaf.org"
+  xmlns:layout="http://www.ultraq.net.nz/thymeleaf/layout"
+>
+<head>
+  <title layout:title-pattern="$LAYOUT_TITLE - $CONTENT_TITLE">
+    Checkout Demo
```

Comment:
Update title to Adyen Giving demo. Note sometimes we hardcode it, sometimes we use placeholders `$LAYOUT_TITLE`. We should make it consistent